### ISSUE TITLE
## Training and Validation Loss (After fixing the overfitting)

### ISSUE THREAD

**Issue by AhmadSabbirChowdhury** [AhmadSabbirChowdhury/Cats-Vs-Dogs-Classification-Kaggle-Problem-Set-#4]
![accurate data 2](https://user-images.githubusercontent.com/45898995/82494383-4b63d200-9b0b-11ea-81e3-997becef8f50.png)
